app/service/data_svc.py

```diff
@@ -489,3 +494,12 @@ def __init__(self, dao):
         _, filename = await self.get_service('file_svc').find_file_path('%s.yml' % pack, location='data')
         for adv in self.strip_yml(filename):
             return [dict(phase=k, id=i) for k, v in adv.get('phases').items() for i in v]
+
+    async def _add_fact_relationships(self, criteria=None):
+        relationships = await self.dao.get('core_relationships', criteria)
+        for r in relationships:
+            r.pop('source')
+            r.pop('link_id')
```

Comment:
Are you popping just to discard the data? If so, why?

Reply:
Correct. I'm removing this data because it is is already stored in the fact dictionary that the relationship list is returned and attached to. The target and edge data is all that's new

Reply:
Maybe this?
```
    async def _add_fact_relationships(self, criteria=None):
        relationships = await self.dao.get('core_relationships', criteria)
        return [dict(edge=r.get('edge'), target=(await self.dao.get('core_fact', dict(id=r.get('target'))))[0])
                for r in relationships if r.get('target')]
```

Reply:
Ok, it is done